src/modules/drop.ts

```diff
@@ -870,20 +870,18 @@ export class DropModule
     proofs: BytesLike[] = [hexZeroPad([0], 32)],
   ): Promise<TransactionReceipt> {
     const claimData = await this.prepareClaim(quantity, proofs);
-    const encoded = [];
-    encoded.push(
-      this.contract.interface.encodeFunctionData("claim", [
-        quantity,
-        claimData.proofs,
-      ]),
-    );
-    encoded.push(
-      this.contract.interface.encodeFunctionData("transferFrom", [
-        await this.getSignerAddress(),
-        addressToClaim,
-        (await this.readOnlyContract.nextTokenIdToMint()).sub(1),
-      ]),
-    );
+    const result = await this.claim(quantity, claimData.proofs);
```

Comment:
this will make it not be a multi-call?

Reply:
Yeah, this is expected. Because we don't know the ID's of the NFTs that are about to be claimed, we need to form the `claim` in one tx, then the transfer calls in a separate (multicall) tx.

@jnsdls